spring-framework-features/spring32/src/main/feature/feature.xml

```diff
@@ -47,7 +47,7 @@
+    <bundle start-level="30">wrap:mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.spring-jms/3.2.18.RELEASE_1$overwrite=merge&amp;Import-Package=javax.jms;version="[1.1,2)", *com.fasterxml.jackson.databind;resolution:=optional;version="[2.0.2,3)", com.fasterxml.jackson.databind.type;resolution:=optional;version="[2.0.2,3)", javax.naming;resolution:=optional, javax.resource;resolution:=optional;version="[1.5,2)", javax.resource.spi;resolution:=optional;version="[1.5,2)", javax.resource.spi.endpoint;resolution:=optional;version="[1.5,2)", javax.xml.transform;resolution:=optional, javax.xml.transform.stream;resolution:=optional, org.aopalliance.intercept;resolution:=optional;version="[1,2)", org.apache.commons.logging;resolution:=optional;version="[1.1.1,2)", com.fasterxml.jackson.databind;resolution:=optional;version="[1.14.1,1.15)", om.fasterxml.jackson.databind.type;resolution:=optional;version="[1.4.2,2)", com.fasterxml.jackson.databind.type;resolution:=optional;version="[1.4.2,2)", org.springframework.aop.framework;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.aop.support;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.beans;version="[3.2.18.RELEASE,3.3)", org.springframework.beans.factory;version="[3.2.18.RELEASE,3.3)", org.springframework.beans.factory.config;version="[3.2.18.RELEASE,3.3)", org.springframework.beans.factory.parsing;version="[3.2.18.RELEASE,3.3)", org.springframework.beans.factory.support;version="[3.2.18.RELEASE,3.3)", org.springframework.beans.factory.xml;version="[3.2.18.RELEASE,3.3)", org.springframework.context;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.core;version="[3.2.18.RELEASE,3.3)", org.springframework.core.task;version="[3.2.18.RELEASE,3.3)", org.springframework.jca.endpoint;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.jndi;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.oxm;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.remoting;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.remoting.support;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.scheduling;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.transaction;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.transaction.support;resolution:=optional;version="[3.2.18.RELEASE,3.3)", org.springframework.util;version="[3.2.18.RELEASE,3.3)", org.w3c.dom;resolution:=optional</bundle>
```

Comment:
There is a duplication of the references to "com.fasterxml.jackson...": the ones that replaced the references to "org.codehaus.jackson..." should be removed (specially because the stated versions are wrong)!